packages/prop-controllers/package.json

```diff
@@ -2,28 +2,29 @@
   "name": "@makeswift/prop-controllers",
   "version": "0.3.0",
   "license": "MIT",
-  "type": "module",
   "files": [
     "dist"
   ],
-  "main": "./dist/index.cjs",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
   "types": "./dist/types/index.d.ts",
   "scripts": {
-    "dev": "concurrently -k 'tsc --watch --preserveWatchOutput' 'tsup src/index.ts --watch'",
+    "dev": "concurrently -k 'tsc --watch --preserveWatchOutput' 'tsup --watch'",
     "clean": "rm -rf dist",
-    "build": "tsc && tsup src/index.ts",
+    "build": "tsc && tsup",
     "prepublishOnly": "turbo run build",
     "test": "jest"
   },
   "dependencies": {
+    "@makeswift/controls": "workspace:*",
```

Comment:
I'm guessing we've already checked that when publishing we're using `pnpm` instead of `npm` and that it's replacing this with the appropriate value?

Reply:
Yep, confirmed with a snapshot release:

![CleanShot 2024-08-22 at 13 29 35@2x](https://github.com/user-attachments/assets/70e4fc8f-951d-45ba-9b71-c7d533e06dfb)

Also confirmed this from a snapshot release installation of the runtime into an actual Next app. We verified that the dependencies on `@makeswift/controls` and `@makeswift/prop-controllers` are correctly replaced.